externs/browser/html5.js

```diff
@@ -2919,7 +2919,7 @@ TimeRanges.prototype.end = function(index) { return 0; };
  * @see http://dev.w3.org/html5/websockets/
  * @constructor
  * @param {string} url
- * @param {string=} opt_protocol
+ * @param {(string|Array<string>)=} opt_protocol
```

Comment:
Please add a `!` (indicating non-nullability):
``` js
 * @param {(string|!Array<string>)=} opt_protocol
```

https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler#nullable-type

Otherwise LGTM.

Reply:
Done and re-pushed as a single commit